### PR TITLE
[combined] Fix typos

### PR DESCRIPTION
**Pull request by xulike666**
I'd like to apology for disturbance to your pr time line from the k8s learning group in cst.zju.edu.cn. This kind of pr from me will combine our commits and keep step with the master branch. It's a great time to stay together with all of you and learn from open source project.

### CODE REVIEW

File: contributors/design-proposals/downward_api_resources_limits_requests.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Downward API for resource limits and requests
+# Downward API for resources limits and requests
```
Comment (liggitt) [inline contributors/design-proposals/downward_api_resources_limits_requests.md]: revert these changes... "resource limits" is correct

File: contributors/devel/api-conventions.md [hunk 2]
```diff
@@ -799,7 +799,7 @@ Kubernetes leverages the concept of *resource versions* to achieve optimistic
 concurrency. All Kubernetes resources have a "resourceVersion" field as part of
 their metadata. This resourceVersion is a string that identifies the internal
 version of an object that can be used by clients to determine when objects have
-changed. When a record is about to be updated, it's version is checked against a
+changed. When a record is about to be updated, it's version that is checked against a
```
Comment (liggitt) [inline contributors/devel/api-conventions.md]: revert this

File: contributors/design-proposals/downward_api_resources_limits_requests.md (hunk 1 above)
Comment (xulike666) [inline contributors/design-proposals/downward_api_resources_limits_requests.md]: In these files we both find "resources limits" and "resource limits" , inconformity or deliberation?

**Comment by SRaddict** [comment]
I signed it!

File: contributors/devel/api-conventions.md (hunk 2 above)
Comment (jonboulle) [inline contributors/devel/api-conventions.md]: it's -> its

File: contributors/design-proposals/downward_api_resources_limits_requests.md (hunk 1 above)
Comment (jonboulle) [inline contributors/design-proposals/downward_api_resources_limits_requests.md]: it should likely always be "resource limits" - the circumstances in which "resources limits" would be appropriate are rare

**Comment by philips** [comment]
I am sorry but there are now conflicts on this commit.

Also, merging commits that include different authors is rather complex. Can this be squashed?

**Comment by xulike666** [comment]
@philips Conflicts have been resolved and thanks for your time to review our phraseology modifications.

File: contributors/devel/on-call-build-cop.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ and [pulls](https://github.com/kubernetes/kubernetes/pulls): Your job is to be
 the first responder to all new issues and PRs. If you are not equipped to do
 this (which is fine!), it is your job to seek guidance!
 
-  * Support issues should be closed and redirected to Stackoverflow (see example
+  * Support issues should be closed and redirected to StackOverflow (see example
```
Comment (jonboulle) [inline contributors/devel/on-call-build-cop.md]: technically it should be Stack Overflow (with space)

File: README.md [hunk 4]
```diff
@@ -68,7 +68,7 @@ The community meeting calendar is available as an [iCal to subscribe to] (https:
 | [UI](sig-ui/README.md) | [@romlein (Dan Romlein, Apprenda)](https://github.com/romlein)<br> [@bryk (Piotr Bryk, Google)](https://github.com/bryk) | [Group](https://groups.google.com/forum/#!forum/kubernetes-sig-ui) | [#sig-ui](https://kubernetes.slack.com/messages/sig-ui/) | Wednesdays at 4:00 PM CEST |
 | [Windows](sig-windows/README.md) | [@michmike77 (Michael Michael, Apprenda)](https://github.com/michmike)| [Group](https://groups.google.com/forum/#!forum/kubernetes-sig-windows) | [#sig-windows](https://kubernetes.slack.com/messages/sig-windows) | Bi-weekly Tuesdays at 10:00 AM PT |
 
-### How to start a SIG
+### How to start an SIG
```
Comment (bgrant0607) [inline README.md]: Ping

File: contributors/design-proposals/access.md [hunk 5]
```diff
@@ -283,7 +283,7 @@ K8s authorization should:
 the system, to integration with existing to enterprise authorization systems.
 - Allow for centralized management of users and policies. In some
 organizations, this will mean that the definition of users and access policies
-needs to reside on a system other than k8s and encompass other web services
+need to reside on a system other than k8s and encompass other web services
```
Comment (bgrant0607) [inline contributors/design-proposals/access.md]: Ping

File: contributors/design-proposals/architecture.md [hunk 6]
```diff
@@ -49,7 +49,7 @@ a unified view of the cluster.
 
 ### `etcd`
 
-All persistent master state is stored in an instance of `etcd`. This provides a
+All persistent master states are stored in an instance of `etcd`. This provides a
```
Comment (bgrant0607) [inline contributors/design-proposals/architecture.md]: Please revert. It's collective.

File: contributors/design-proposals/enhance-pluggable-policy.md [hunk 7]
```diff
@@ -75,7 +75,7 @@ namespace" (see [ResourceAccessReview](#ResourceAccessReview) further down).
 ```go
 // OLD
 type Authorizer interface {
-  Authorize(a Attributes) error
+  Authorize(an Attributes) error
```
Comment (bgrant0607) [inline contributors/design-proposals/enhance-pluggable-policy.md]: Please revert. This is code.

File: contributors/design-proposals/enhance-pluggable-policy.md [hunk 8]
```diff
@@ -89,7 +89,7 @@ type Authorizer interface {
   // reason is an optional return value that can describe why a policy decision
   //   was made.  Reasons are useful during debugging when trying to figure out
   //   why a user or group has access to perform a particular action.
-  Authorize(ctx api.Context, a Attributes) (allowed bool, reason string, evaluationError error)
+  Authorize(ctx api.Context, an Attributes) (allowed bool, reason string, evaluationError error)
```
Comment (bgrant0607) [inline contributors/design-proposals/enhance-pluggable-policy.md]: Code again

File: contributors/design-proposals/enhance-pluggable-policy.md [hunk 9]
```diff
@@ -103,7 +103,7 @@ type AuthorizerIntrospection interface {
   // Attributes to determine which users and groups are allowed to perform the
   // described action in the namespace. This API enables the ResourceBasedReview
   // requests below
-  GetAllowedSubjects(ctx api.Context, a Attributes) (users util.StringSet, groups util.StringSet, evaluationError error)
+  GetAllowedSubjects(ctx api.Context, an Attributes) (users util.StringSet, groups util.StringSet, evaluationError error)
```
Comment (bgrant0607) [inline contributors/design-proposals/enhance-pluggable-policy.md]: Code again

File: contributors/devel/adding-an-APIGroup.md [hunk 10]
```diff
@@ -33,7 +33,7 @@ pkg/client/unversioned}/import_known_versions.go, if you want to make your group
 accessible to other packages in the kube-apiserver binary, binaries that uses
 the client package.
 
-Step 2 and 3 are mechanical, we plan on autogenerate these using the
+Step 2 and 3 are mechanical, we plan on autogenerating these using the
```
Comment (bgrant0607) [inline contributors/devel/adding-an-APIGroup.md]: I suggest "to autogenerate" instead

File: contributors/devel/e2e-node-tests.md [hunk 11]
```diff
@@ -106,7 +106,7 @@ make test-e2e-node REMOTE=true IMAGES="<comma-separated-list-images>"
 
 ## Run tests against a running GCE instance (not an image)
 
-This is useful if you have an host instance running already and want to run the tests there instead of on a new instance.
+This is useful if you have a host instance running already and want to run the tests there instead of on a new instance.
```
Comment (bgrant0607) [inline contributors/devel/e2e-node-tests.md]: Technically, "an" is correct, but I don't care one way or the other.

File: contributors/devel/update-release-docs.md [hunk 12]
```diff
@@ -4,7 +4,7 @@
 
 - [Table of Contents](#table-of-contents)
 - [Overview](#overview)
-- [Adding a new docs collection for a release](#adding-a-new-docs-collection-for-a-release)
+- [Adding new docs collection for a release](#adding-a-new-docs-collection-for-a-release)
```
Comment (bgrant0607) [inline contributors/devel/update-release-docs.md]: Ping

File: contributors/devel/update-release-docs.md [hunk 13]
```diff
@@ -21,10 +21,10 @@ branch](https://github.com/kubernetes/kubernetes/tree/gh-pages) of kubernetes re
 Updating docs in that branch will update http://kubernetes.io
 
 There are 2 scenarios which require updating docs:
-* Adding a new docs collection for a release.
+* Adding new docs collection for a release.
```
Comment (bgrant0607) [inline contributors/devel/update-release-docs.md]: Ping

File: contributors/devel/update-release-docs.md [hunk 14]
```diff
@@ -21,10 +21,10 @@ branch](https://github.com/kubernetes/kubernetes/tree/gh-pages) of kubernetes re
...
 * Updating docs in an existing collection.
 
-# Adding a new docs collection for a release
+# Adding new docs collection for a release
```
Comment (bgrant0607) [inline contributors/devel/update-release-docs.md]: Ping

File: contributors/devel/updating-docs-for-feature-changes.md [hunk 15]
```diff
@@ -60,7 +60,7 @@ Anyone making user facing changes to kubernetes.  This is especially important f
 ## Who should help review docs changes?
 
 * cc *@kubernetes/docs*
-* Changes to [kubernetes/docs](https://github.com/kubernetes/kubernetes.github.io) repo must have both a Technical Review and a Docs Review
+* Changes to [kubernetes/docs](https://github.com/kubernetes/kubernetes.github.io) repo must have both a Technical Review and Docs Review
```
Comment (bgrant0607) [inline contributors/devel/updating-docs-for-feature-changes.md]: Ping

File: governance.md [hunk 16]
```diff
@@ -54,7 +54,7 @@ Code and documentation contributors:
   that frequent contributors will assist in our code-review process and with project
   maintenance
 - **Org Member**: an active contributor for at least 3 months; active enough to be useful
-  to assign issues to them and add them to a github team (e.g., for a SIG) for notification
+  to assign issues to them and add them to a github team (e.g., for an SIG) for notification
```
Comment (bgrant0607) [inline governance.md]: ping

**Comment by philips** [comment]
I am sorry. Doing spelling fixes in this way is really chewing up a lot of everyone's time on code reviews.

What I would suggest is that you target a single word that isn't an or a and fix it across all files in the tree. This is often what happens on the Linux kernel and it has two advantages:

1) Code review is really easy
2) It is way harder to make a mistake in changing the wrong thing.

I am going to close this PR for now as I don't think it is an effective use of everyone's time given the number of commits, number of errors, and amount of rebasing.

**Pull request closed without merging**